src/cli.rs

```diff
@@ -46,6 +46,7 @@ pub fn build_cli() -> App<'static, 'static> {
                         .short("o")
                         .long("output-dir")
                         .takes_value(true)
+                        .default_value("$PROJECT_ROOT/public")
                         .help("Outputs the generated site in the given path"),
```

Comment:
Or may be a change like this:
```
.help("Outputs the generated site in the given path (by default directory named public in project root)"),
```

Reply:
I would prefer that change

Reply:
I made the suggested change further shortening the string.